app/models/role_loader.rb

```diff
@@ -0,0 +1,45 @@
+class RoleLoader
+
+  def available_roles_for_application(name)
```

Comment:
Ok, I see. Maybe we could actually just convert it to `roles_available_for_application?` or something like that. Which would just find if there's at least one and wouldn't have to construct the `Role` models, etc? Or are we actually using the roles list themselves somehow?

I'm just trying to think of a case when we could expose all roles and then mix and match them across organisation types, which would then be a mistake. If that makes sense.